packages/icons/readme.md

```diff
@@ -0,0 +1,117 @@
+# Cypress Icons
```

Comment:
@elevatebart These readme instructions are out of date considering this is an internal package. Can you update the Readme to accurately reflect it's usage?

Reply:
Fixed the readme, would appreciate a proof read.